Makefile

```diff
@@ -21,12 +21,43 @@ endif
 
 include $(BOLOS_SDK)/Makefile.defines
 
-APPNAME      = "Boilerplate"
+########################################
+#        Mandatory configuration       #
+########################################
+# Application name
+APPNAME = "Boilerplate"
```

Comment:
Do we have any limitation on the app name such as allowed characters or even maximum size ? If so it could be good to document it here